ДЗ.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Седона</title>
+</head>
+
+<body>
+
+<hedar>
+ <nav>
+  <a>
+      <img src="img/welcome.png" width="456.65" height="350.88" alt="Логотип «Седона»">
+    </a>
+   
+  <ul>
+      <li><a href="blank.html">Информация</a></li>
+      <li><a href="photos and videos.html">фото и Видео</a></li>
+      <li><a href="the map of the state.html">Карта штата</a></li>
+      <li><a href="hotels.html">Гостиницы</a></li>
+    </ul>
+ </nav>
+</hedar>
+<main>
+ <!-- <h1> Седона</h1>-->
+  <h2>Седона-небольшой городок в Аризоне,заслуживающий большего!</h2>
+   <p>Рассмотрим 5 причин,по которым Седона круче,чем Гранд Каньон!</p>
+   <sektion>
+   <h3>Преимущества</h3>
+     <ul>
+      <li>
+	  <h2>Настоящий городок</h2>
+	    <a>
+      <img src="img/-№1-.png" width="51" height="21" alt="-№1-">
+    </a>
+        <p>Седона-не аттракцион для туристов,там течет своя жизнь</p>
+		</li>
+		<li>
+        <a>
+      <img src="img/icon-1.png" width="75" height="72" alt="icona-1">
+    </a>
+	     <h2>Жильё</h2>
+	     <p>Рекомендуем пожить в настоящем мотеле,все как в кино!</p>
+		 </li>
+		 <li>
+		   <a>
+      <img src="img/icon-3.png" width="75" height="70" alt="icon-3">
+    </a>
+	      <h2>Еда</h2>
+		  <p>Всегда заказывайте фирменный бургер,вы не разочаруетесь!</p>
+		  </li>
+		  <li>
+		   <a>
+      <img src="img/icon-2.png" width="64" height="77" alt="icon-2">
+    </a>
+	       <h2>Сувениры</h2>
+		   <p>Не только китайского но и местного производства!</p>
+      </li>
+	  <li>
+	  <h2>Там есть мост дьявола</h2>
+	   <a>
+      <img src="img/-№2-.png" width="51" height="21" alt="-№2-">
+    </a>
+	   <p>Да,по нему можно пройти! Если конечно вы осмелитесь </p>
+</li>
+  <li>
+      <h2>Небольшая площадь</h2>
+	   <a>
+      <img src="img/-№3-.png" width="51" height="21" alt="-№3-">
+    </a>
+	<p>Все достопримечательности находятся очень близко</p>
+     </li>
+      <li>
+       <h2>Красивая дорога</h2>
+	    <a>
+      <img src="img/-№4-.png" width="51" height="21" alt="-№4-">
+    </a>
+	<p>Ехать в Седону из Лас-Вегаса совсем не скучьно!</p>
+    </li>
+    <li>
+     <h2>Мало туристов</h2>
+	  <a>
+      <img src="img/-№5-.png" width="51" height="21" alt="-№5-">
+    </a>
+	 <p>Большинство едет в Гранд Каньон и толпится там</p>
+	 </li>
+    </ul>
+  </sektion>
+  <sektion>
+   <h2>Заинтересовались?</h2>
+   <p>Укажите предпологаемые даты поездки,и мы покажем вам лучшие предложения гостинец в Седоне</p>
+  </sektion>
+  <sektion>
+  <h2>Поиск гостиницы в Седоне</h2>
+  <!--форма поиска гостиницы-->
+  </sektion>
```

Comment:
Это единая секция с формой. + "Поиск гостиницы в Седоне" это интерактивный элемент, в данном случае ссылка
```suggestion
  <section>
   <h2>Заинтересовались?</h2>
   <p>Укажите предпологаемые даты поездки,и мы покажем вам лучшие предложения гостинец в Седоне</p>
   <a href="">Поиск гостиницы в Седоне</a>
</section>
```